model/adjuster/span_merger.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2018 The Jaeger Authors.
```

Comment:
there should be no UI merge in this PR - probably needs a rebase off the master.